src/tests/client.js

```diff
@@ -0,0 +1,10 @@
+import ApolloClient from 'apollo-client'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import { SchemaLink } from 'apollo-link-schema'
+
+import schema from '@api/schema'
+
+export default new ApolloClient({
+  cache: new InMemoryCache(),
```

Comment:
Do we really have to pass that option? @rmnbrd 
We’ll probably not use it for anything else so if we can do without it, let’s remove the dependency.